### PR TITLE
Tracking: Documentation

### PR DESCRIPTION
**Pull request by pfraze**
Adding documentation to help developers enter this project.

### CODE REVIEW

**Comment by dominictarr** [comment]
:+1:

**Comment by pfraze** [comment]
@dominictarr @mixmix any suggestions (other docs, changes to these docs)?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Contributing
```
Comment (mixmix) [inline CONTRIBUTING.md]: I like what you've written. I want to note a couple of the terms surprised me. 
e.g. when I went to read Project-Structure, I was expecting a broader scope than code-structure. 

I wonder what the best place is for talking about the team / organisation structure and building in ways wider than code ?
E.g. are designs / mockups welcome / useful.

Comment (pfraze) [inline CONTRIBUTING.md]: I've started the following etherpads:

 - [ui process proposals](http://grimwire.com:9001/p/ui-process).
 - [feature proposals](http://grimwire.com:9001/p/pw-feature-ideas).

let's use these as a starting point

**Pull request merged**